README.rst

```diff
@@ -184,8 +185,8 @@ Updating objects with progress logging
 --------------------------------------
 
 Since an upgrade step often updates a set of objects indexed in the catalog,
-there is a useful helper method combining querying the catalog with the
-`ProgressLogger` (see below).
+there is a useful helper method `self.objects()` which combines querying the
+catalog with the `ProgressLogger` (see below).
```

Comment:
as you're improving the readme, can you use a link instead of above/below here? (only if easily targetable via e.g. title).

Reply:
Sure!